index.js

```diff
@@ -1,4 +1,5 @@
 const { createServer } = require('./lib/server');
 
 const port = process.env.PORT;
-createServer(port);
+const host = process.env.HOST || '0.0.0.0';
```

Comment:
`localhost` is the safer option if someone is running this on their laptop directly. Let's use `localhost` if the env var is not specified.

But since it's not really useful for docker containers to expose servers only on `localhost`, I think it makes sense to default to `0.0.0.0` there.

Reply:
🤔didn't think of docker stuffs, good to know!